docassemble/AssemblyLine/data/questions/ql_baseline.yml

```diff
@@ -536,6 +536,81 @@ fields:
     code: countries_list()
     default: ${ AL_DEFAULT_COUNTRY }
 ---
+generic object: ALIndividual
+id: any service address
+question: |
+  At what address will you serve ${ x }?
+fields:
+  - ${ x }'s service address is: x.service_address
+    datatype: object_radio
+    choices:
+      - x.address if defined(x.address.attr_name("address")) else None
+      - x.mailing_address if defined(x.mailing_address.attr_name("address")) else None
+    object labeler: |
+      lambda y: y.on_one_line()      
+    none of the above: |
+      Somewhere else
+    disable others: True      
+  - ${ x.service_address.address_label }: x.service_address.address
+    address autocomplete: True
+  - ${ x.service_address.unit_label }: x.service_address.unit
+    required: False
+  - ${ x.service_address.city_label }: x.service_address.city
+  - ${ x.service_address.state_label }: x.service_address.state
+    code: |
+      states_list(country_code=AL_DEFAULT_COUNTRY)
+    default: ${ AL_DEFAULT_STATE }
+  - ${ x.service_address.zip_label }: x.service_address.zip
+    required: False      
+  - ${ x.service_address.country_label }: x.service_address.country
+    code: countries_list()
+    default: ${ AL_DEFAULT_COUNTRY }
+---
+generic object: ALIndividual
+id: service method
+question: |
+  How will you get a copy of the documents to ${ x }?
+fields:
+  - I will serve ${ x } by: x.service_method
+    input type: radio
+    code: al_service_methods
+---
+comment: |
+  You can customize this block, and also add a translation for it with a language modifier,
+  demonstrated below
+variable name: al_service_methods
+use objects: True
+data:
+  e-filing system: 
+    question: E-filing system
+    document: e-filing system
+  mail:
+    question: Mail
+    document: mail
+  hand:
+    question: In-hand delivery
+    document: in-hand
```

Comment:
Is "in-hand" the right way of saying this? Motion to stay eviction just uses "hand-delivery", both on the document and in the question, it seems slightly easier to read